### ISSUE TITLE
Patching out loadStorage breaks influxql cumulative_sum tests in influxdb

### ISSUE THREAD

**Issue by rockstar** [influxdata/flux#3145]
Recently, we changed the implementation of `loadStorage` to validate the data that's being loaded. To work around the (prevalent) issue of existing bad data, we patched out `loadStorage` in many tests to bypass that validation, with the intent to come back and fix the data in places where it needed to be fixed.

`testing/influxql/cumulative_sum`, for instance, needs to be fixed, because the test data is used by influx itself.

**Comment by wolffcm** [comment]
@rockstar Is the issue here that the test fails with something like this?
```
      --- FAIL: TestFluxEndToEnd/testing/promql/extrapolatedRate_counter_rate (0.02s)
            end_to_end_test.go:226: @8:1-13:2: option "testing.loadStorage" reassigned
```
I have seen something similar in tests I'm running in OSS.

**Comment by rockstar** [comment]
Yeah, that’s the exact issue. Looking at the tests themselves, we can’t be overriding `loadStorage` for those tests because those tests need to set their own implementation.

**Comment by wolffcm** [comment]
Should this be merged with #3155 then?
